### ISSUE TITLE
Add custom flag to allow deploying without building

### ISSUE THREAD

**Issue by JDurstberger** [seek-oss/serverless-haskell#97]
This is just an idea / feature request. 

I have been playing around with serverless-haskell for a few days now and it works quite well. 
The only issue I can see is using it with continuous deployment. 

I use Gitlab CI and I would love to be able to split building and deploying into 2 build steps. 
This would require being able to invoke `serverless deploy` without building, but using an already built binary. 

So basically I would love to be able to 

1. Build without docker
2. Deploy binary built in step 1

Of course there might be a better way of doing this, it was just an idea I've had. 
I am also willing to help building this if you think it is a good idea.

**Comment by koterpillar** [comment]
Thank you for the suggestion, sorry for the delay getting back to you.

If you can find a way to pass a custom flag to Serverless, make a PR and I'll be happy to include it.
